packages/synapse-react-client/src/components/SageResourcesPopover/SageResourcesPopover.tsx

```diff
@@ -58,107 +51,98 @@ export function SageResourcesPopover({
     sourceAppConfigTableID,
     additionalFilters,
   )
+
   return (
-    <>
-      <Popover
-        open={open}
-        anchorEl={anchorEl}
-        onClose={onClose}
-        sx={{ width: '90%', height: '90%' }}
-        anchorOrigin={{
-          vertical: 'bottom',
-          horizontal: 'center',
-        }}
-        transformOrigin={{
-          vertical: 'top',
-          horizontal: 'center',
-        }}
-      >
-        <Grid container spacing={2} sx={{ p: '24px' }}>
-          <Grid item xs={12} lg={3}>
-            <Box sx={{ p: theme.spacing(4), pr: '5px' }}>
-              <Typography
-                variant="headline1"
-                sx={{ fontSize: '40px', pb: '30px', fontWeight: 600 }}
+    <Popover
+      open={open}
+      anchorEl={anchorEl}
+      onClose={onClose}
+      sx={{ width: '90%', height: '90%' }}
+      anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+      transformOrigin={{ vertical: 'top', horizontal: 'center' }}
+    >
+      <Grid container spacing={2} sx={{ p: '24px' }}>
+        <Grid size={{ xs: 12, lg: 3 }}>
+          <Box sx={{ p: theme.spacing(4), pr: '5px' }}>
+            <Typography
+              variant="headline1"
+              sx={{ fontSize: '30px', pb: '30px', fontWeight: 600 }}
+            >
+              {resourceName}
+            </Typography>
+            <Typography variant="body1" sx={{ pb: '10px', fontWeight: 500 }}>
+              {description}
+            </Typography>
+            {allResourcesUrl && (
+              <Button
+                type="button"
+                color="primary"
+                variant="contained"
+                sx={{
+                  p: '10px 50px',
+                  mt: '30px',
+                  height: '100%',
+                  '&:hover': { color: 'white' },
+                }}
+                href={allResourcesUrl}
               >
-                {resourceName}
-              </Typography>
-              <Typography variant="body1" sx={{ pb: '10px', fontWeight: 500 }}>
-                {description}
-              </Typography>
-              {allResourcesUrl && (
-                <Button
-                  type="button"
-                  color="primary"
-                  variant="contained"
+                See all {resourceName}
+              </Button>
+            )}
+          </Box>
+        </Grid>
+
+        <Grid size={{ xs: 12, lg: 9 }}>
+          <Grid container>
+            {sourceAppConfigs
+              ?.filter(config => {
+                if (config.appURL.toLowerCase().startsWith('http')) {
+                  const url = new URL(config.appURL)
+                  return url.hostname !== hostname
+                }
+                return true
+              })
+              .map(config => (
+                <Grid
+                  key={config.appId}
+                  size={{ xs: 12, sm: 6, lg: 4 }} // ⬅️ here too
```

Comment:
```suggestion
                  size={{ xs: 12, sm: 6, lg: 4 }}
```

Agent telling you where it made changes? 😉

Reply:
Yep!